great_expectations/experimental/datasources/interfaces.py

```diff
@@ -305,38 +296,11 @@ class Datasource(
 
     # class attrs
     asset_types: ClassVar[List[Type[DataAsset]]] = []
-    # Datasource instance attrs but these will be fed into the `execution_engine` constructor
-    _excluded_eng_args: ClassVar[Set[str]] = {
-        "name",
-        "type",
-        "execution_engine",
-        "assets",
-    }
```

Comment:
By removing this we now force each `DataAsset`  definition to manually instantiate the execution engine.
We are adding extra steps to defining a `DataSource` with no obvious benefit (that I can see).

Reply:
As discussed in our check-in I will be adding this back.

Reply:
✅